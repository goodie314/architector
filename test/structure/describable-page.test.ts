import DescribablePage from '../../src/structure/describable-page';
import SimpleHtmlComponent from '../components/test-components/simple-html-component';

describe('Describable page module', () => {
    test('successfully render app in the document body', () => {
        new DescribablePage(new SimpleHtmlComponent().compose()).render();

        expect(document.body.childNodes.length).toEqual(1);
        expect(document.body.firstElementChild).toMatchSnapshot();
    });
});
