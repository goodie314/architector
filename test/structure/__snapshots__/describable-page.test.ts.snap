// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Describable page module successfully render app in the document body 1`] = `
<div
  class="class-1 class-2"
  id="test-id"
  test="test-attribute"
>
  Hello
</div>
`;
